components/CreateNav.js

```diff
@@ -0,0 +1,40 @@
+import Link from "next/link";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+
+const StyledNav = styled.nav`
+  display: flex;
+  justify-content: center;
+  gap: 24px;
+`;
+
+const StyledActiveLink = styled(Link)`
+  font-weight: 800;
+  text-decoration: none;
+  color: #000000;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: #000000;
+`;
+
+export default function CreateNav() {
+  const router = useRouter();
+  return (
+    <StyledNav>
+      {router.pathname === "/create-card" ? (
+        <StyledActiveLink href={"/create-card"}>Card</StyledActiveLink>
+      ) : (
+        <StyledLink href={"/create-card"}>Card</StyledLink>
+      )}
```

Comment:
See my comment above.

```suggestion
         <StyledLink href="/create-card" $isActive={router.pathname === "/create-card"} >
        Card
      </StyledLink>
```